plugins/chain_api_plugin/chain.swagger.yaml

```diff
@@ -89,6 +89,171 @@ paths:
+                        items:
+                          type: object
+                          description: Description of a finalizer
+                          properties:
+                            'description':
+                              type: string
+                              description: Name of finalizer, typically a producer account
+                              example: finalizerone
+                            weight:
+                              type: integer
+                              description: The weight of this finalizer used in determine when threshold of votes is reached
+                              example: 1
+                            public_key:
+                              type: string
+                              description: The BLS Public Key used by this finalizer
+                              example: "PUB_BLS_PcJG7lYgwZcsFGe-i4WWYDw1GtP7r5vGWFcRe97TVcp5f5_h6RRRPRhZSGKtRF4CtdJxAP50VewH4VIhMIPfhsu9C4GGH4MgOzSwl51N3EK6XiAiRHQ_lcGvxggipYwIReHkhQ"
+                  pending_finalizer_policy:
+                    type: object
+                    description: List of pending finalizers
+                    properties:
+                      generation:
+                        type: integer
+                        description: generation number of finalizer policy
+                        example: 2
+                      threshold:
+                        type: integer
+                        description: total weight needed to advance finality
+                        example: 15
+                      finalizers:
+                        type: array
+                        items:
+                          type: object
+                          description: Description of a finalizer
+                          properties:
+                            'description':
+                              type: string
+                              description: Name of finalizer, typically a producer account
+                              example: finalizerone
+                            weight:
+                              type: integer
+                              description: The weight of this finalizer used in determine when threshold of votes is reached
+                              example: 1
+                            public_key:
+                              type: string
+                              description: The BLS Public Key used by this finalizer
+                              example: "PUB_BLS_PcJG7lYgwZcsFGe-i4WWYDw1GtP7r5vGWFcRe97TVcp5f5_h6RRRPRhZSGKtRF4CtdJxAP50VewH4VIhMIPfhsu9C4GGH4MgOzSwl51N3EK6XiAiRHQ_lcGvxggipYwIReHkhQ"
+                  last_tracked_votes:
+                    type: array
+                    description: Record of last votes for the finalizers in current active finalizer policy and pending finalizer policy.
+                    items:
+                      type: object
+                      properties:
+                        public_key:
```

Comment:
`description` is now the first field, `public_key` is the second. The order of the rest is correct.